src/async_impl/request.rs

```diff
@@ -276,6 +293,17 @@ impl RequestBuilder {
         self
     }
 
+    /// Enables a response body limit.
+    ///
+    /// The response body limits the size of the response. Responses larger than
+    /// the limit will be rejected. Only this request is affected. The limit is specified in bytes.
+    pub fn response_body_limit(mut self, response_body_limit: usize) -> RequestBuilder {
```

Comment:
Probably should be a `u64`. On 32-bit systems, `usize` would mean the maximum body size would be 4GB, and it's not uncommon for things to be bigger than that.